src/constants.ts

```diff
@@ -10,6 +10,7 @@ export const GLOBAL_CONF_VERSION = '1';
 export const REGIONS_API_URL: { [key: string]: string } = {
   iran: 'https://api.iran.liara.ir',
   germany: 'https://api.liara.ir',
+  mail: 'https://mail-service.iran.liara.ir',
```

Comment:
Please extract this to its own constant.

```suggestion
  mail: 'https://mail-service.iran.liara.ir',
```